packages/forma-36-codemod/transforms/v5/icons.js

```diff
@@ -74,8 +74,8 @@ const iconsMap = {
   Image: 'ImageSquare',
   InfoCircle: 'Info',
   Language: 'Globe',
-  Link: 'LinkSimpleHorizontal',
-  LinkAlternate: 'LinkSimpleHorizontal',
+  Link: 'LinkSimple',
+  LinkAlternate: 'LinkSimple',
   ListBulleted: 'ListBullets',
```

Comment:
Do we need to map any of the old ones to the `LinkBreak`??

Reply:
We did not have one before 🙅